src/cloud/internal/cloudservice.cpp

```diff
@@ -78,9 +79,12 @@ void CloudService::init()
     m_oauth2->setAuthorizationUrl(configuration()->authorizationUrl());
     m_oauth2->setAccessTokenUrl(configuration()->accessTokenUrl());
     m_oauth2->setModifyParametersFunction([](QAbstractOAuth::Stage stage, QVariantMap* parameters) {
-        if (stage == QAbstractOAuth::Stage::RequestingAuthorization) {
-            (*parameters).insert(EDITOR_SOURCE_KEY, EDITOR_SOURCE_VALUE);
-        }
+        (*parameters).insert(EDITOR_SOURCE_KEY, EDITOR_SOURCE_VALUE);
```

Comment:
What's the reason for writing `(*parameters).insert` instead of `parameters->insert`?

Reply:
Because it was that way before too I guess